app/src/ui/history/commit-summary.tsx

```diff
@@ -401,6 +393,48 @@ export class CommitSummary extends React.Component<
     clipboard.writeText(this.props.commit.sha)
   }
 
+  private renderChangedFilesDescription = () => {
+    const fileCount = this.props.changesetData.files.length
+    const filesPlural = fileCount === 1 ? 'file' : 'files'
+    const filesShortDescription = `${fileCount} changed ${filesPlural}`
+
+    let filesAdded = 0
+    let filesModified = 0
+    let filesRemoved = 0
+    for (const file of this.props.changesetData.files) {
+      switch (file.status.kind) {
+        case AppFileStatusKind.New:
+          filesAdded += 1
+          break
+        case AppFileStatusKind.Modified:
+          filesModified += 1
+          break
+        case AppFileStatusKind.Deleted:
+          filesRemoved += 1
+          break
+      }
+    }
+
+    const filesLongDescription = (
+      <>
+        {filesAdded ? <li>{filesAdded} added</li> : null}
```

Comment:
We prefer explicit checks - (same applies to next two lines)

```suggestion
        {filesAdded > 0 ? <li>{filesAdded} added</li> : null}
```